src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckJavaAgent.kt

```diff
@@ -388,6 +389,8 @@ internal object LincheckClassFileTransformer : ClassFileTransformer {
         internalClassName: String,
         classBytes: ByteArray
     ): ByteArray = transformedClassesCache.computeIfAbsent(internalClassName.canonicalClassName) {
+        LincheckLogger.log("Transform class '${internalClassName.canonicalClassName}'")
```

Comment:
Let's use `debug` level for instrumentation logging.

Reply:
Also no need to convert to `canonicalClassName` here, lets print class name as is.